readme.txt

```diff
@@ -37,7 +37,7 @@ Fixed a bug where it may attempt to load the Block Editor for post types that do
 Updated for WordPress 5.0.
 Changed all "Gutenberg" names/references to "Block Editor".
 Refreshed the settings UI.
-Removed disabling of the Gutenberg plugin. This was added for testing in WordPress 4.9. Users who want to continue following the development of Gutenberg in WordPress 5.0 and beyond will not need another plugin to disable it.
+Removed disabling of the Gutenberg plugin. This was added for testing in WordPress 4.9+ Users who want to continue following the development of Gutenberg in WordPress 5.0 and beyond will not need another plugin to disable it.

```

Comment:
`4.9+` is not that clear, thinking most people will understand `version 4.9` though :)

Reply:
OK